src/index.html

```diff
@@ -11,7 +11,110 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+
+      <fieldset>
+        <legend>Personal information</legend>
+        <label class="block"> Surname:
+          <input
+          type="text"
+          name="text"
+          minlength="3"
+          maxlength="20"
+          required
+          >
+        </label>
+        <label class="persInform"> Name:
+          <input
+          type="text"
+          name="text"
+          required
+          >
+        </label>
```

Comment:
```suggestion
        <label class="persInform"> 
          Name:
          <input
            type="text"
            name="text"
            required
          >
        </label>
```

please pay attention to the formatting 
check checklist p2, p3 and fix formatting everywhere